subprojects/distributions-integ-tests/src/integTest/groovy/org/gradle/DocsDistributionIntegrationSpec.groovy

```diff
@@ -30,6 +30,11 @@ class DocsDistributionIntegrationSpec extends DistributionIntegrationSpec {
         "docs"
     }
 
+    @Override
+    int getMaxDistributionSizeBytes() {
+        return 37 * 1024 * 1024
```

Comment:
Is there a point in checking the distro sizes for the docs and sources? It gives us more things to update but I don't see why we need this.

An alternative would be to have a list of sizes declared in `DistributionIntegrationSpec` for the binaries, the sources and the docs, and these methods would return things like `DOCS_SIZE` (for the `docs` distro) or `BIN_SIZE + DOCS_SIZE + SOURCES_SIZE` (for the `all` distro).